src/pages/home/sidebar/OptionRow.js

```diff
@@ -128,11 +138,22 @@ const OptionRow = ({
                                 && (
                                     <MultipleAvatars
                                         avatarImageURLs={option.icons}
-                                        optionIsFocused={optionIsFocused}
                                         size={mode === 'compact' ? 'small' : 'default'}
-                                        secondAvatarStyle={hovered && !optionIsFocused
-                                            ? styles.secondAvatarHovered
-                                            : undefined}
+                                        secondAvatarStyle={[
```

Comment:
@marcaaron I have moved all the Style logic to the parent. This should be like this as the `optionIsFocused` prop is irrelevant to the `MultipleAvatars`. It was used only for OptionRow Context. Now we have another state hovered so I think it better to just pass the style prop whose only purpose is to style the second Avatar(it should be clear to anyone working with it). We manage all the states outside `MultipleAvatar`. Also, It's important to manage it here as we have to get the background color from the parent

Reply:
I think this is a better approach. But now we have the problem of passing style objects inline and should probably make this a utility method stored in `styles.js` to maintain consistency with the rest of the app.